docs/BR.md

```diff
@@ -1224,27 +1224,41 @@ No stipulation.
+3. The CA obtains evidence that the Subscriber's Private Key corresponding to the Public Key in the Certificate suffered a Key Compromise (CRLReason #1, keyCompromise);
+4. The CA is made aware of a demonstrated or proven method that can easily compute the Subscriber's Private Key based on the Public Key in the Certificate (such as a Debian weak key, see <https://wiki.debian.org/SSLkeys>) (CRLReason #1, keyCompromise);
+5. The CA obtains evidence that the validation of domain authorization or control for any Fully-Qualified Domain Name or IP address in the Certificate should not be relied upon (CRLReason #4, superseded).
+
+The CA SHOULD revoke a certificate within 24 hours and MUST revoke a Certificate within 5 days and use the corresponding CRLReason if one or more of the following occurs:
+
+1. The Certificate no longer complies with the requirements of [Section 6.1.5](#615-key-sizes) and [Section 6.1.6](#616-public-key-parameters-generation-and-quality-checking) (CRLReason #4, superseded);
+2. The CA obtains evidence that the Certificate was misused (CRLReason #9, privilegeWithdrawn);
+3. The CA is made aware that a Subscriber has violated one or more of its material obligations under the Subscriber Agreement or Terms of Use (CRLReason #9, privilegeWithdrawn);
+4. The CA is made aware of any circumstance indicating that use of a Fully-Qualified Domain Name or IP address in the Certificate is no longer legally permitted (e.g. a court or arbitrator has revoked a Domain Name Registrant's right to use the Domain Name, a relevant licensing or services agreement between the Domain Name Registrant and the Applicant has terminated, or the Domain Name Registrant has failed to renew the Domain Name) (CRLReason #5, cessationOfOperation);
+5. The CA is made aware that a Wildcard Certificate has been used to authenticate a fraudulently misleading subordinate Fully-Qualified Domain Name (CRLReason #9, privilegeWithdrawn);
+6. The CA is made aware of a material change in the information contained in the Certificate (CRLReason #9, privilegeWithdrawn);
+7. The CA is made aware that the Certificate was not issued in accordance with these Requirements or the CA's Certificate Policy or Certification Practice Statement (CRLReason #4, superseded);
+8. The CA determines or is made aware that any of the information appearing in the Certificate is inaccurate (CRLReason #9, privilegeWithdrawn);
+9. The CA's right to issue Certificates under these Requirements expires or is revoked or terminated, unless the CA has made arrangements to continue maintaining the CRL/OCSP Repository (CRLReason "unspecified (0)" which results in no reasonCode extension being provided in the CRL);
+10. Revocation, other than what is described in this section 4.9.1.1, is required by the CA's Certificate Policy and/or Certification Practice Statement (CRLReason "unspecified (0)" which results in no reasonCode extension being provided in the CRL); or
```

Comment:
Should this be rewritten to be clearer? I understand the intent of "other than what is described in this section 4.9.1.1", but other readers might not understand that required revocation under a CP/CPS might be for reason #4 (superseded) or #9 (privilegeWithdrawn).

Reply:
I think this should read, "10.	Revocation is required by the CA's Certificate Policy and/or Certification Practice Statement **_for a reason that is not otherwise described in this section 4.9.1.1_** ..."

Reply:
Or, "10. Revocation is required by the CA's Certificate Policy and/or Certification Practice Statement _**for a reason that is not otherwise required to be specified by this section 4.9.1.1**_ ..."

Reply:
Resolved in https://github.com/cabforum/servercert/pull/405/commits/0a07e046326101ef3b57572daebd3cf45ff4840f